### DISCUSSION TITLE
Manage users with API

### DISCUSSION THREAD

**Discussion by max-allan-surevine** [DefectDojo/django-DefectDojo#4806] [Q&A]
Hi, I can add users with the v2 API but I can't change their groups or reset passwords, etc. That is managed by Django (I think!). 

Are there any docs/guidelines on how to auth to django with my API token and what endpoints are available etc.??
(I'd like to set a user password and add to a group, with a curl command from the CLI.) 

I can see the request to "/admin/auth/user/abc/change/" with "groups: xyz" as form data (where xyz and abc are group/user IDs) Do I need to specify all the other fields I'm not changing? Can I use JSON? etc. )

**Comment by valentijnscholten** [comment]
I think by default there is no API for the Django admin part. It would need something like this https://github.com/amirasaran/django-restful-admin. Had no updates for over a year, so not sure if it is working/maintained.

**Comment by StefanFl** [comment]
The django groups have no effect for the user interface and only limited effect for the API. You should use the DefectDojo groups instead, see https://defectdojo.github.io/django-DefectDojo/usage/permissions/
